src/roadmapper/milestone.py

```diff
@@ -59,6 +58,38 @@ def draw(self, painter: Painter) -> None:
                 self.diamond_height,
                 self.fill_colour,
             )
+
+        alignment = (
+            self.text_alignment.lower() if self.text_alignment is not None else None
+        )
+
+        if alignment is None or alignment == "centre":
```

Comment:
Do you also want to consider the case of `centre:offset_value`?

Reply:
This is still a good comment and I'll have to look at how it applies now that I have moved the logic for parsing alignment out to it's own class.

Reply:
I added  validation for this since it seems bad to silently ignore the input of an offset with a center alignment.

https://github.com/csgoh/roadmapper/commit/a6af49d308528d91f7f478866adc919beee2d7fc